catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraGenres.scala

```diff
@@ -0,0 +1,28 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import uk.ac.wellcome.models.work.internal.{Concept, Genre, Period, Place}
+import uk.ac.wellcome.transformer.source.SierraBibData
+
+trait SierraGenres extends MarcUtils {
+
+  def getGenres(bibData: SierraBibData): List[Genre] = {
+    getGenresForMarcTag(bibData, "655")
+  }
+
+  private def getGenresForMarcTag(bibData: SierraBibData, marcTag: String) = {
```

Comment:
We routinely add the described transformation as a comment here. See: https://github.com/wellcometrust/platform/blob/787ac7d009a7e611feb69288716bbc157dfcfd42/catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraDescription.scala#L25